tasks/main.yml

```diff
@@ -3,12 +3,34 @@
 - name: Update rs_collector configuration
   template: src="etc/rs-collector.conf.j2" dest="{{ _rs_collector.config_file|default("/etc/rs-collector.conf") }}" mode="0640" owner=root group=root
 
-- name: Check if init (override) file for rs-collector exists
-  stat:
-    path: /etc/init/rs-collector.override
-  register: stat_rs_collector_override
-  when: _rs_collector.oom_score_adj is defined
-
-- name: Set oom score for rs-collector
-  lineinfile: dest="{{ stat_rs_collector_override.stat.path }}" state=present regexp='^oom score' line="oom score {{ _rs_collector.oom_score_adj }}"
-  when: _rs_collector.oom_score_adj is defined and stat_rs_collector_override.stat.exists
+- block:
+    - name: Check if init (override) file for rs-collector exists
+      stat:
+        path: /etc/init/rs-collector.override
+      register: stat_rs_collector_override
+      when: _rs_collector.oom_score_adj is defined
+
+    - name: Set oom score for rs-collector in Upstart
```

Comment:
Wieso diese Leerzeile danach?

Reply:
durch die Verschiebung in den block Task sieht das grafisch im PR so aus.